dev/test_scrape_article.py

```diff
@@ -0,0 +1,264 @@
+import datetime
+import random
+import time
+
+import pandas as pd
+import psycopg2
+import requests
+from bs4 import BeautifulSoup
+
+
+# Function to request and parse via BeautifulSoup
+def request_and_parse(url):
+    time.sleep(random.random())  # Add a random waiting time (less than 1 sec) for each request
```

Comment:
We might have a library for managing the timeout -- I would have to look more into this later.